### DISCUSSION TITLE
I can't upload files using Requests

### DISCUSSION THREAD

**Discussion by marcoarildo** [fastapi/fastapi#8597] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
-- Server Code
-- 

from typing import List
from fastapi import FastAPI, File, UploadFile
from fastapi.responses import HTMLResponse

app = FastAPI()

@app.post("/files/")
async def create_files(files: List[bytes] = File(...)):
    return {"file_sizes": [len(file) for file in files]}


@app.post("/uploadfiles/")
async def create_upload_files(files: List[UploadFile]):
    return {"filenames": [file.filename for file in files]}


@app.get("/")
async def main():
    content = """
<body>
<form action="/files/" enctype="multipart/form-data" method="post">
<input name="files" type="file" multiple>
<input type="submit">
</form>
<form action="/uploadfiles/" enctype="multipart/form-data" method="post">
<input name="files" type="file" multiple>
<input type="submit">
</form>
</body>
    """
    return HTMLResponse(content=content)


-- client code
-- 
import requests

url = 'http://127.0.0.1:8000/files/'

files = {'file': open('imagem1.jpg','rb')}
headers = {'Content-Type': 'multipart/form-data'}

r = requests.post(url, headers = headers, files=files)

print(r.status_code)
print(r.text)
```

### Description

When I do tests using the Browser or via the Curl command they work correctly.
But when I consume the api using python (requests) it keeps giving an error.

I already imported the python-multipart library.

And when run it returns:

**400
{"detail":"There was an error parsing the body"}**

I would be very grateful if anyone can help.

Sorry for my basic english

Tks

### Operating System

macOS

### FastAPI Version

0.73.0

### Python Version

Python 3.8.5

**Comment by yinziyan1206** [comment, marked as answer]
remove the headers, change keyword "file" to "files"
